lib/trino-parquet/src/main/java/io/trino/parquet/predicate/PredicateUtils.java

```diff
@@ -126,7 +140,7 @@ public static Predicate buildPredicate(
                 columnReferences.add(descriptor);
             }
         }
-        return new TupleDomainParquetPredicate(parquetTupleDomain, columnReferences.build(), timeZone);
+        return new TupleDomainParquetPredicate(parquetTupleDomain, columnReferences.build(), timeZone, domainCompactionThreshold);
     }
 
     public static boolean predicateMatches(
```

Comment:
Please remove this method and the DEFAULT_DOMAIN_COMPACTION_THRESHOLD constant

Reply:
done.